ui/store/action-queue/index.ts

```diff
@@ -24,10 +23,25 @@ import { isManifestV3 } from '../../../shared/modules/mv3.utils';
 //   }, {});
 // }
 
-let background = null;
-let promisifiedBackground = null;
+let background:
+  | ({
```

Comment:
Does it make sense to create variables for these types or they just aren't used anywhere else?

Reply:
It may make sense if we find them used elsewhere but as of right now they aren't.